packages/aws-fargate/images/README.md

```diff
@@ -0,0 +1,91 @@
+Node.js Fargate Container Images
```

Comment:
I moved the README one level up from packages/aws-fargate/images/instana-aws-fargate/README.md (and gave it a bit of an overhaul)